### ISSUE TITLE
How to run the specified model training and validation code？

### ISSUE THREAD

**Issue by ZChengLong578** [nasib-ullah/video-captioning-models-in-Pytorch#4]
I don't use Jupyter Notebook much and don't know how to run _ipynb_ files using commands. So, can you give examples of commands that run training code and validation code? Examples are training and validation commands for the MARN model.

**Comment by nasib-ullah** [comment]
Hi, you can create a py file (e.g main.py) with the content of the ipynb file and just run that python file from the terminal (python main.py). For changing the model and settings just change the config.py file (according to your need) before running main.py. I hope this answers your question.

**Comment by ZChengLong578** [comment]
Thank you for your reply. I tried to run the module code of MARN in.py file, and I found that you annotated part of the code, including the attenuation of learning rate and the output of verification indicators. I'm having a hard time deciding which code is required for training and which is required for testing. Also, I found no code to save the best models during training. Can you give a sample code for MARN training and testing? Thank you very much!
